docs/frontend_plan.md

```diff
@@ -4,6 +4,7 @@
 This document captures the actionable plan for evolving the Orcheo canvas from its current developer-centric prototype into a production-ready, user-centered experience. It consolidates design and engineering activities so product, design, and frontend contributors can execute in lockstep while staying aligned with backend capabilities.
 
 ## Current Baseline Assessment
+- **Migrated canvas foundation**: The legacy canvas now runs on the refreshed React Flow surface with minimap, snapping, and chat affordances, but undo/redo history, duplication/export actions, and search remain unimplemented.
```

Comment:
This migration was not for refreshing the surface, but more about connecting the existing Canvas to the new backend